### ISSUE TITLE
11.2.1 --> 11.1.2 with blobs??  i6P

### ISSUE THREAD

**Issue by J0113** [encounter/futurerestore#22]
Hello,

Can I downgrade from iOS 11.2.1 to 11.1.2 with blobs that I've made with 'tsssaver.1conan.com' using the nonce collision method? (On iPhone 6s Plus)

Thanks Jolle

**Comment by Stoppels** [comment]
No, you _basically_ need a jailbreak to downgrade **from** 11.2+. Collisions have been patched since iOS 10.2.1/10.3.
